lrc_file/LrcFile.py

```diff
@@ -66,10 +74,23 @@ def _get_cpu_metrics(lnst_run: RecipeRun, evaluated_cpu_metrics: list[str]) -> d
     }
 
 
+def _get_tc_metrics(lnst_run: RecipeRun, evaluated_tc_metrics: list[str]) -> dict[str, float]:
+    return {
+        f"{i}_{key}": value
+        for i, result in enumerate(lnst_run.results)
+        if _is_tc_measurement_result(result)
+        for key, value in result.data.items()
+        if key in evaluated_tc_metrics
+    }
+
 def _get_cpu_data(lnst_run: RecipeRun) -> list[Run]:
     try:
         m1_results, m2_results = filter(_is_cpu_measurement_result, lnst_run.results)
     except ValueError:
+        #TODO Figure out a better way to handle this TrafficControlRecipe not having cpu info
+        from lnst.Recipes.ENRT import TrafficControlRecipe
```

Comment:
I'd move it to the top of the file, as pep8 says, it's best practise. Performance gain is almost zero

Reply:
fixed